spec/models/category_spec.rb

```diff
@@ -27,5 +27,44 @@
 
       expect(category.errors[:title].first).to eq('muss ausgefüllt werden')
     end
+
+    it 'should have a unique position' do
+      category = Category.new
+      category.position = 100
+      category.valid?
+
+      expect(category.errors[:position].first).to eq('ist bereits vergeben')
+    end
+  end
+
+  context 'orders' do
+    it 'orders all_parents correctly' do
+      category = Category.new
+      category.title = 'Diverses'
+      category.position = 300
+      category.parent = nil
+      category.save!
+
+      categories = Category.all_parents
+      expect(categories.first).to eq(categories(:'software-engineering'))
+      expect(categories.second).to eq(categories(:'system-engineering'))
+      expect(categories.third).to eq(category)
+    end
+
+    it 'orders all children correctly' do
+      categories = Category.all_children
+      expect(categories.first).to eq(categories(:ruby))
+      expect(categories.second).to eq(categories(:java))
+      expect(categories.third).to eq(categories(:'linux-engineering'))
+    end
+
+    it 'orders all skills correctly' do
+      categories = Category.list
```

Comment:
category_titles = Category.list.pluck(:title)
expect(category_titles).to eq(['Bla', 'Bli', ...])